test/WorkItemBuiltins/get_global_id-non-uniform.cl

```diff
@@ -0,0 +1,34 @@
+// RUN: clspv -cl-std=CL2.0 -inline-entry-points -descriptormap=%t.dmap %s -o %t.spv
+// RUN: spirv-dis -o %t2.spvasm %t.spv
+// RUN: FileCheck %s < %t2.spvasm
+// RUN: FileCheck --check-prefix=DMAP %s < %t.dmap
+// RUN: spirv-val --target-env vulkan1.0 %t.spv
+
+// DMAP: pushconstant,name,region_offset,offset,0,size,12
+
+// CHECK:     OpMemberDecorate %[[_struct_3:[0-9a-zA-Z_]+]] 0 Offset 0
+// CHECK:     OpDecorate %[[_struct_3]] Block
+// CHECK:     OpMemberDecorate %[[_struct_12:[0-9a-zA-Z_]+]] 0 Offset 0
+// CHECK:     OpDecorate %[[_struct_12]] Block
```

Comment:
I would just drop these. The push constant tests should provide coverage that the right decorations are generated. Same edit for most of the tests to reduce overall LOC.

Reply:
Yes, good idea. It'll reduce the amount of changes needed in the future as well. Now done.